test/commands/force/org/scratchOrgCreate.test.ts

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Aliases, Config, SfdxError, Messages, Org, SfdxProject } from '@salesforce/core';
+import { fromStub, stubInterface, stubMethod } from '@salesforce/ts-sinon';
+import { IConfig } from '@oclif/config';
+import * as sinon from 'sinon';
+import { expect } from '@salesforce/command/lib/test';
+import { UX } from '@salesforce/command';
+import { assert } from 'sinon';
+import { Create } from '../../../../src/commands/force/org/beta/create';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-org', 'create');
+
+describe('org:create', () => {
+  const sandbox = sinon.createSandbox();
+  const oclifConfigStub = fromStub(stubInterface<IConfig>(sandbox));
+  const clientSecret = '123456';
+  // stubs
+  let resolveProjectConfigStub: sinon.SinonStub;
+  let scratchOrgCreateStub: sinon.SinonStub;
+  let uxLogStub: sinon.SinonStub;
+  let uxWarnStub: sinon.SinonStub;
+  let uxErrorStub: sinon.SinonStub;
+  let promptStub: sinon.SinonStub;
+  let cmd: TestCreate;
+
+  class TestCreate extends Create {
+    public async runIt() {
+      await this.init();
+      return this.run();
+    }
+    public setOrg(org: Org) {
+      this.org = org;
+    }
+    public setProject(project: SfdxProject) {
+      this.project = project;
+    }
+  }
+
+  const createCommand = async (params: string[]) => {
```

Comment:
Unless I'm missing the async part of this function it looks like (as written) you can remove the `async`.  Since it doesn't call `cmd.runIt()` it's synchronous.

Reply:
@WillieRuemmele I copy this code from you maybe you want to take a look at the `createCommand` for sandbox and also remove the unnecessary `async`.